spec/lib/importers/revision_score_importer_spec.rb

```diff
@@ -57,22 +57,6 @@
     end
   end
 
-  it 'saves wp10 scores by article' do
-    VCR.use_cassette 'revision_scores/by_article' do
-      described_class.update_revision_scores_for_all_wikis
```

Comment:
Removed all the specs calling `update_revision_scores_for_all_wikis` because removed the method.